src/config/scrivitoSites.ts

```diff
@@ -1,8 +1,19 @@
-import { Obj, currentSiteId, getInstanceId, load, navigateTo } from 'scrivito'
+import {
+  Link,
+  Obj,
+  currentSiteId,
+  getInstanceId,
+  load,
+  navigateTo,
+  urlFor,
+} from 'scrivito'
 import { isMultitenancyEnabled } from './scrivitoTenants'
 
 const location = typeof window !== 'undefined' ? window.location : undefined
 
+const rootContentId: string =
+  import.meta.env.SCRIVITO_ROOT_CONTENT_ID || 'c2a0aab78be05a4e'
```

Comment:
How about
```suggestion
const rootContentId =
  ensureString(import.meta.env.SCRIVITO_ROOT_CONTENT_ID) || 'c2a0aab78be05a4e'
```

This way we ensure, that this is an actual string and not just claim it is.

Reply:
I went with what we get from `import.meta.env`. This should be good enough for an equals comparison.